extensions-contrib/moving-average-query/src/main/java/org/apache/druid/query/movingaverage/MovingAverageQueryModule.java

```diff
@@ -58,4 +68,25 @@ public void configure(Binder binder)
                                                  )));
   }
 
+  @Provides
+  @LazySingleton
+  @Named("movingAverage")
+  public QuerySegmentWalker createQueryWalker(@Self Set<NodeRole> nodeRoles, Injector injector)
+  {
+    if (!nodeRoles.contains(NodeRole.BROKER)) {
```

Comment:
Have you tried using a moving average query through the router? Does it still work?

When I tried the extension a little while ago, I had to enable it on the router as well to get the query to run successfully.

Reply:
Yes, I tested the current fix with quickstart which contains all worker types. Tested the following on quickstart : 
1. Clean startup on all workers
2. Simple ingestion job
3. Simple queries 
4. Moving average queries as mentioned in https://druid.apache.org/docs/latest/development/extensions-contrib/moving-average-query.html
Please let me know incase anything else would be useful as well

Reply:
This is great! Thanks @rohangarg